### DISCUSSION TITLE
error while loading shared libraries: libnvinfer.so.6

### DISCUSSION THREAD

**Discussion by gubinjie** [PaddlePaddle/PaddleOCR#12787] [Q&A]
版本号/Version：Paddle：  PaddleOCR2.6

Project templates used: [https://aistudio.baidu.com/projectdetail/5942251?channelType=0&channel=0](url)

The following exception occurred：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/37869445/e0c0a515-5919-4f6d-b7b7-738dbb98752e)

How to solve it？TH

**Comment by UserWangZz** [comment]
确认你已经安装了TensorRT，并且它是针对你的系统架构（如x86_64）的正确版本。

如果已安装，确保libnvinfer.so.6所在的目录包含在你的库路径中。你可以通过设置LD_LIBRARY_PATH环境变量来添加库路径，例如：

export LD_LIBRARY_PATH=/path/to/tensorrt/lib:$LD_LIBRARY_PATH
